R/load.R

```diff
@@ -757,11 +756,11 @@ renv_load_report_project <- function(project) {
   version <- renv_metadata_version_friendly()
 
   if (length(profile)) {
-    fmt <- "* (%s) Project '%s' loaded. [renv %s]"
-    writef(fmt, profile, renv_path_aliased(project), version)
+    fmt <- "- Project '%s' loaded with %s profile."
```

Comment:
Do we want to make a decision around whether we use `*` prefixes or `-` prefixes for messages like these? Or maybe we should start wrapping these into internal calls to cli-like functions like `inform()` so we can easily adapt in the future?

Reply:
I thought we were moving towards `-` everywhere?

Reply:
Did we make that decision explicitly? I'm happy to do it but can't recall if we consciously chose to.

Reply:
I don't think we discussed it but I think I assumed based on your recent PRs 😄